Dockerfile.test.cpu

```diff
@@ -113,7 +113,11 @@ RUN if [[ ${PYTORCH_PACKAGE} == "torch-nightly" ]]; then \
 RUN pip install "Pillow<7.0" --no-deps
 
 # Install MXNet.
-RUN pip install ${MXNET_PACKAGE}
+RUN if [[ ${MXNET_PACKAGE} == "mxnet-nightly" ]]; then \
+        pip install -pre mxnet -f https://dist.mxnet.io/python/all; \
```

Comment:
On this line, it should be `--pre` instead of `-pre`.

Reply:
Fixed.